examples/multiphysics/static-bubble-chns/outputs/R_0.40/static-bubble-chns.prm

```diff
@@ -0,0 +1,164 @@
+# Listing of Parameters
```

Comment:
Add license. Do you need all of these pre-made folders or you think it's better to generate them on the fly ?%